docs/integrations/sources/hubspot.md

```diff
@@ -59,23 +59,35 @@ Depending on the type of engagement, different properties will be set for that o
 
 **Note**: HubSpot API currently only supports `quotes` endpoint using API Key, using Oauth it is impossible to access this stream (as reported by [community.hubspot.com](https://community.hubspot.com/t5/APIs-Integrations/Help-with-using-Feedback-CRM-API-and-Quotes-CRM-API/m-p/449104/highlight/true#M44411)).
 
-## Getting Started \(Airbyte Open-Source / Airbyte Cloud\)
+## Getting Started
```

Comment:
In our input configuration, we link to this section for the Client ID, Client Secret, and Refresh Token - but we don't list those as requirements here nor do we explicitly mention how to retrieve these fields. Can we please add them?

Reply:
Yes, added a link to oauth quickstart https://github.com/airbytehq/airbyte/pull/11218/files#diff-5faad53afb0b7795a4e1bd2b62eda2424f5756d8f08c2514db84be2977533ccdR82 and https://github.com/airbytehq/airbyte/pull/11218/files#diff-5faad53afb0b7795a4e1bd2b62eda2424f5756d8f08c2514db84be2977533ccdR96